package/gluon-config-mode-contact-info/files/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -0,0 +1,35 @@
+local cbi = require "luci.cbi"
```

Comment:
this whole file is in the wrong folder, a while ago it switched places, maybe you started your development based on an older revision of gluon.